elasticdl/python/ps/parameters.py

```diff
@@ -157,3 +170,23 @@ def create_slot_params(self, slot_names, init_values):
                     init_values[slot_name],
                     True,
                 )
+
+    def export_to_model_pb(self):
+        """ Export all parameters including embedding and non-embedding
+        parameters to `elasticdl_pb2.Model` which can be serialized.
+        """
+        pb_model = elasticdl_pb2.Model()
+        for name, var in self.non_embedding_params.items():
+            emplace_tensor_pb_from_ndarray(
+                pb_model.param, var.numpy(), name=name
+            )
+
+        for name, embedding_table in self.embedding_params.items():
+            embedding_table_tensor = embedding_table.to_tensor()
+            tensor_pb = pb_model.param.add()
+            serialize_tensor(embedding_table_tensor, tensor_pb)
+
+            embedding_info = embedding_table.to_embedding_table_info_pb()
+            pb_model.embedding_table_info.append(embedding_info)
+
```

Comment:
1 pb_model --> model_pb
2 version should also be serialized, model_pb.version = self.version